modin/pandas/test/test_dataframe.py

```diff
@@ -3456,29 +3407,26 @@ def test_prod(self, request, data, axis, skipna, numeric_only, min_count):
     @pytest.mark.parametrize(
         "min_count", int_arg_values, ids=arg_keys("min_count", int_arg_keys)
     )
-    def test_product(self, request, data, axis, skipna, numeric_only, min_count):
-        modin_df = pd.DataFrame(data)
-        pandas_df = pandas.DataFrame(data)
-
-        try:
-            pandas_result = pandas_df.product(
-                axis=axis, skipna=skipna, numeric_only=numeric_only, min_count=min_count
-            )
-        except Exception:
-            with pytest.raises(TypeError):
-                modin_df.product(
-                    axis=axis,
-                    skipna=skipna,
-                    numeric_only=numeric_only,
-                    min_count=min_count,
-                )
-        else:
-            modin_result = modin_df.product(
-                axis=axis, skipna=skipna, numeric_only=numeric_only, min_count=min_count
-            )
-            df_equals(modin_result, pandas_result)
+    @pytest.mark.parametrize("is_transposed", [False, True])
```

Comment:
Why you add the option?

Reply:
Previous test implementation computed `prod/sum` for transposed and not transposed data, I just moved that into test parameter